undefined reference to `MULE_EVT_LOGLINE'

Hello,
as reported at http://forum.amule.org/index.php?topic=20801.0 , I needed to apply the below patch to avoid this error (after I regenerated the files running autogen.sh):

```
g++ -W -Wall -Wshadow -Wundef -pthread  -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -Wall -lpthread -Wl,-z,relro -o amulecmd amulecmd-TextClient.o amulecmd-DataToText.o amulecmd-ExternalConnector.o amulecmd-LoggerConsole.o amulecmd-OtherFunctions.o amulecmd-NetworkFunctions.o  -Llibs/common -Llibs/ec/cpp -L. -lmulecommon -lec -lmulesocket -L/usr/lib/x86_64-linux-gnu -pthread   -lwx_baseu_net-3.0 -lwx_baseu-3.0  -lreadline   -lz    
amulecmd-LoggerConsole.o: In function `CLoggingEvent':
/home/morph/deb/build-area/amule-2.3.1+git20151120.0023527bc2/obj-x86_64-linux-gnu/src/../../src/Logger.h:346: undefined reference to `MULE_EVT_LOGLINE'
collect2: error: ld returned 1 exit status
Makefile:1219: recipe for target 'amulecmd' failed
```

```
--- a/src/LoggerConsole.cpp
+++ b/src/LoggerConsole.cpp
@@ -29,6 +29,7 @@
 
 #include "Logger.h"                            // for Add(Debug)LogLineM()
 
+DEFINE_LOCAL_EVENT_TYPE(MULE_EVT_LOGLINE)
 
 #ifdef __DEBUG__
 bool CLogger::IsEnabled(DebugType /*type*/) const
```

--- a/src/LoggerConsole.cpp
+++ b/src/LoggerConsole.cpp
@@ -29,6 +29,7 @@
 
 #include "Logger.h"				// for Add(Debug)LogLineM()
 
+DEFINE_LOCAL_EVENT_TYPE(MULE_EVT_LOGLINE)
 
 #ifdef __DEBUG__
 bool CLogger::IsEnabled(DebugType /*type*/) const

